Project.toml

```diff
@@ -4,12 +4,13 @@ authors = ["Humans of Julia"]
 version = "0.1.0"
 
 [deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
```

Comment:
I don't think these are used?

Reply:
You are right. As of today, we are not yet using either package. I deleted them